projects/frontend/data-pipelines/gui/.editorconfig

```diff
@@ -18,6 +21,9 @@ max_line_length = off
 quote_type = double
 indent_size = 2
 
+[*.{scss, css}]
+max_line_length = 140
+
```

Comment:
I don't see any recommendation for __line_length__ in the coding standard, but I set it to 140 because previously we set 140 characters in shared-components and to make them consistent, that's why I added this file here which is copy from shared-components with only one difference in 
```
[*.{ts, html}]
max_line_length = 80
```
and that's intentionally in order not to introduce too much noise in this PR, but in separate PR to increase it to 140 characters which will make both libraries with same format configuration

default length in SC and Taurus was 140 characters.
in prettier default configuration is 80 characters which is too short IMO

Reply:
I agree, shorter line lengths are outdated due todays hardware capacity, allowing people to observe on the screen longer lines. Also agree we need to be consistent. 
Have you considered adding an issue to revise and plan this in todo?

Reply:
I'll create an issue to make it consistent with incrementing to 140 characters in a separate PR :)

Reply:
issue created: https://github.com/vmware/versatile-data-kit/issues/1801